### PR TITLE
Add Google Podcast subscribe link

### PR DESCRIPTION
**Pull request by stephaniehobson**
Closes https://github.com/mozilla/bedrock/issues/5826

### CODE REVIEW

File: static/images/subscribe-badge-google.svg [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="134px" height="34px" viewBox="0 0 134 34" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
+    <title>F00_listen_on_badge</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
```
Comment (jpetto) [inline static/images/subscribe-badge-google.svg]: Nit - run this through [SVGOMG](https://jakearchibald.github.io/svgomg/) to remove cruft.

**Comment by jpetto** [comment]
The layout is a little busted right at/around 760px:

screen shot 2018-06-20 at 4 03 06 pm

Maybe we set a width on that list and let the badges wrap?

File: static/images/subscribe-badge-google.svg (hunk 1 above)
Comment (stephaniehobson) [reply static/images/subscribe-badge-google.svg]: I'm surprised Google didn't do this themselves >_<

**Comment by stephaniehobson** [comment]
That took longer than it needed to. Can we have container queries already?

Updated ⏯

**Pull request merged**